Git/commit.sublime-syntax

```diff
@@ -0,0 +1,84 @@
+%YAML 1.2
+---
+# Highlight regular git commits, merge commits, and tags.
+
+name: Git Commit
+file_extensions:
+  - COMMIT_EDITMSG
+  - MERGE_MSG
+  - TAG_EDITMSG
+  - syntax_test_git_commit
+  - syntax_test_git_merge
+  - syntax_test_git_tag
+scope: source.gitcommit
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - include: commit-message
+
+  branch-line:
+    - match: \bOn branch\s+(.*)
+      captures:
+        1: support.type.branch-name.gitcommit
+
+  change:
```

Comment:
This is an interesting context, but can be improved. For example, highlight the `++++++------` lines accordingly. Take a look at how [GitSavvy](https://github.com/divmain/GitSavvy/blob/7b885da26b7effc1c622ac66911dea123c61c2d4/syntax/show_commit_diffstat.sublime-syntax#L9-L16) does it.